cognite_toolkit/_cdf_tk/apps/_dump_app.py

```diff
@@ -517,3 +523,81 @@ def dump_timeseries_cmd(
                 verbose,
             )
         )
+
+    @staticmethod
+    def dump_event_cmd(
+        ctx: typer.Context,
+        hierarchy: Annotated[
+            Optional[list[str]],
+            typer.Option(
+                "--hierarchy",
+                "-h",
+                help="Asset hierarchy (sub-trees) to dump event from.",
+            ),
+        ] = None,
+        data_set: Annotated[
+            Optional[list[str]],
+            typer.Option(
+                "--data-set",
+                "-d",
+                help="Data set to dump. If neither hierarchy nor data set is provided, the user will be prompted.",
+            ),
+        ] = None,
+        format_: Annotated[
+            str,
+            typer.Option(
+                "--format",
+                "-f",
+                help="Format to dump the event in. Supported formats: csv, and parquet.",
+            ),
+        ] = "csv",
+        limit: Annotated[
+            Optional[int],
+            typer.Option(
+                "--limit",
+                "-l",
+                help="Limit the number of events to dump.",
+            ),
+        ] = None,
+        output_dir: Annotated[
+            Path,
+            typer.Option(
+                "--output-dir",
+                "-o",
+                help="Where to dump the events files.",
+                allow_dash=True,
+            ),
+        ] = Path("tmp"),
+        clean: Annotated[
+            bool,
+            typer.Option(
+                "--clean",
+                "-c",
+                help="Delete the output directory before dumping the events.",
+            ),
+        ] = False,
+        verbose: Annotated[
+            bool,
+            typer.Option(
+                "--verbose",
+                "-v",
+                help="Turn on to get more verbose output when running the command",
+            ),
+        ] = False,
+    ) -> None:
+        """This command will dump the selected events to the selected format in the folder specified, defaults to /tmp."""
+        cmd = DumpDataCommand()
+        cmd.validate_directory(output_dir, clean)
+        client = EnvironmentVariables.create_from_environment().get_client()
+        if hierarchy is None and data_set is None:
+            hierarchy, data_set = EventInteractiveSelect(client).interactive_select_hierarchy_datasets()
+        cmd.run(
+            lambda: cmd.dump_table(
+                EventFinder(client, hierarchy or [], data_set or []),
+                output_dir,
+                clean,
+                limit,
+                format_,  # type: ignore [arg-type]
```

Comment:
Suggestion: You could avoid this type: ignore if you shifted the `is_supported_format` method up here. You could even give it a convenient signature like `validate_format(str) -> Literal['csv', 'parquet']`. It also looks to me like this would technically allow `yaml` as format, but that isn't documented anywhere?